exchanges/sharedtestvalues/sharedtestvalues.go

```diff
@@ -147,3 +150,38 @@ func ForceFileStandard(t *testing.T, pattern string) error {
 	}
 	return nil
 }
+
+// TestFixtureToDataHandler takes a new empty exchange and configures a new websocket handler for it, and squirts the json path contents to it
+// It accepts a reader function, which is probably e.wsHandleData but could be anything
+func TestFixtureToDataHandler(t *testing.T, seed, e exchange.IBotExchange, fixturePath string, reader func([]byte) error) {
+	b := e.GetBase()
+	p, err := seed.GetBase().CurrencyPairs.Clone()
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	b.CurrencyPairs = p //nolint:govet // copylocks not relevant; We just cloned it
```

Comment:
```go
	newBase := e.GetBase()
	seedBase := seed.GetBase()

	newBase.CurrencyPairs.RequestFormat = seedBase.CurrencyPairs.RequestFormat
	newBase.CurrencyPairs.ConfigFormat = seedBase.CurrencyPairs.ConfigFormat
	newBase.CurrencyPairs.UseGlobalFormat = seedBase.CurrencyPairs.UseGlobalFormat
	newBase.CurrencyPairs.Pairs = seedBase.CurrencyPairs.Pairs
```

Given this is for a test, I don't think `Clone` as a func is necessary and can only just copy what is relevant. It also removes nolints and copying of locks

Reply:
Sure, I think that makes sense.

Reply:
Irony: The reason I went to use Clone was because I reasoned we should lock the seed base to copy it.
And then forgot to do so!

Fixed.